src/components/Checkbox/checkbox.module.css

```diff
@@ -0,0 +1,46 @@
+/* Checkbox */
+
+
+.checkbox {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 1.6em;
+  height: 1.6em;
+  border-radius: 7px;
```

Comment:
You are repeating these properties. Instead you can reuse them, like so: 

```
.checkbox,
.checkbox:checked,
.checkbox:hover {
  -webkit-appearance: none;
  appearance: none;
  width: 1.6em;
  height: 1.6em;
  border-radius: 8px; 
  outline: none;
  cursor: pointer;
  margin-right: 20px;
}
```

Then below you can add the specific properties to each class